package-lock.json

```diff
@@ -14,12 +14,12 @@
       }
     },
     "node_modules/@babel/code-frame": {
-      "version": "7.14.5",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.14.5.tgz",
-      "integrity": "sha512-9pzDqyc6OLDaqe+zbACgFkb6fKMNG6CObKpnYXChRsvYGyEdc7CA2BaqeOM+vOtCS5ndmJicPJhKAwYRI6UfFw==",
+      "version": "7.16.0",
```

Comment:
Not a bad idea, but I don't see how the lockfile changes relate to the formatting

Reply:
See https://github.com/nodejs/TSC/pull/1115#discussion_r744423106. The update of the dependencies means a certain class of auto-fix peculiarities no longer happen.